src/Application.php

```diff
@@ -154,9 +154,6 @@ public function refineUriSelection($cwd)
             return;
         }
         $selfAliasRecord = $this->aliasManager->getSelf();
-        if (!$selfAliasRecord->hasRoot()) {
```

Comment:
Do we maybe need to still exist if there is neither a site record root nor a preflight-discovered root? I'm unsure of the impact; I'm fine with just removing this if there are no problems in that scenario.